src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Formatting/CSharpOnTypeFormattingPass.cs

```diff
@@ -117,7 +117,13 @@ public async override Task<FormattingResult> ExecuteAsync(FormattingContext cont
         var filteredEdits = FilterCSharpTextEdits(context, mappedEdits);
         if (filteredEdits.Length == 0)
         {
-            // There are no CSharp edits for us to apply. No op.
+            // There are no CSharp edits for us to apply, but we might still need to check for using statements, or other
```

Comment:
"or other edits outside of our mappable area"

This isn't true, because we're currently checking if using is being added right?

Reply:
It kinda is, it kinda isn't. There might be other edits outside our mappable area _and_ usings being added, even if we don't end up caring about those other edits. But having said that, I guess the wording "we still need to check" does make it wrong.

But more than that, this is me being a little cheeky and forward thinking, because I copied the comment from my branch that allows us to handle new methods being generated by Roslyn too :)

Reply:
Reworded